firefly/kernel/memory-manager/virtual/virtual.cpp

```diff
@@ -11,20 +11,32 @@ using namespace firefly::mm::relocation::conversion;
 using namespace mm;
 
 constexpr size_t GB = 0x40000000UL;
+constexpr size_t PAGE_SIZE = 4096;
 
-VirtualMemoryManager::VirtualMemoryManager(bool initial_mapping) {
+VirtualMemoryManager::VirtualMemoryManager(bool initial_mapping, stivale2_struct_tag_memmap *mmap) {
     if (initial_mapping)
```

Comment:
if (!initial) {
    return;
}
configure();

Reply:
dw about virtual.cpp, I'm going to change everything so this change isn't worth it